.gitignore

```diff
@@ -6,4 +6,5 @@
 # Specific files/dirs to ignore for this pipeline
 */tmp/*
 */out/*
-/_targets/*
\ No newline at end of file
+*/log/*
```

Comment:
I think the instructions wanted you to not ignore the log file output so you can track if that changes.

Reply:
I will update this before merging.